src/routes/PaymentNotices.tsx

```diff
@@ -23,17 +23,15 @@ const paymentNoticesMock: Array<CardProps & { id: number }> = [
     },
     id: 2,
     paymentInfo: 'TARI 2024',
-    amount: '171,00 €',
-    multiPayment: true
```

Comment:
should be enough to set false multiPayment rather than deleting completely the functionality